command/agent/testdata/template.hcl

```diff
@@ -0,0 +1,37 @@
+client {
```

Comment:
You'll need to run `hclfmt` on this file before the linter passes so that tests can be run.

Reply:
Cool. Let me do that now

Reply:
done